topology/probes/netlink.go

```diff
@@ -689,6 +690,10 @@ Ready:
 						}
 						tr.Metadata["Statistics"] = u.statsToMap(stats)
 						lastMetrics := u.statsToMap(&metric)
+						if reflect.DeepEqual(tr.Metadata["Statistics"], lastMetrics) {
```

Comment:
oops, you are correct i am comparing the wrong values. I have changed the code as you suggested with a small improvement. thanks!